src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -140,7 +140,7 @@ private void AddInputOutput(MamlInputOutput io)
                 return;
             }
 
-            var extraNewLine = string.IsNullOrEmpty(io.Description) || ShouldBreak(io.FormatOption);
+            var extraNewLine = ShouldBreak(io.FormatOption);
```

Comment:
Can you provide rationality behind this change?

Reply:
In the new method of generating, I was getting more new lines then necessary (two new lines between each type).  I thought this unnecessary, and the new generation method always happens when io.Description is empty, so I made is so new lines are only added when the shouldbreak returns true.